src/config.py

```diff
@@ -27,6 +27,8 @@
 VOLUME_MULTIPLIER = 0.5
 # The maximum number of songs to download concurrently
 MAXIMUM_CONCURRENT_DOWNLOADS = 8
+# Holds a default URL value jump url for jump_url parameter in embed_song.
+DEFAULT_JUMP_URL = "https://www.youtube.com/watch?v=dQw4w9WgXcQ"
```

Comment:
calling it "default" isn't really accurate, this is the jump url used for previews. it's not really default, if anything it's the non-default since the default is the jump url to the real message